css_profile/css_profile_enm.js

```diff
@@ -2,9 +2,9 @@ if (typeof (dojo) != "undefined") {
     require(["dojo/domReady!"], function () {
         try {
             dojo.place(
-                '<link rel="stylesheet" type="text/css" href="/files/customizer/css_profile/css_profile_enm.css\"></link>',
-                dojo.doc.head,
-                'last'
+               "<link rel=\"stylesheet\" type=\"text/css\" href=\"/files/customizer/profiles/css_profile_enm.css?repoName=homepage-customization\"></link>",

```

Comment:
You need to change the folder name fro css_profile to profiles. I don't see that change as part of this commit.